server/src/main/java/com/kritsit/casetracker/server/datalayer/VehicleRepository.java

```diff
@@ -44,4 +44,23 @@ public VehicleRepository(IPersistenceService db){
             throw new RowToModelParseException("Error retrieving vehicles from database for defendant name: " + defendant.getName());
         }
     }
+
+	@Override
```

Comment:
@Override not necessary as the class is implementing interface and not overriding a superclass